### DISCUSSION TITLE
[Release Vote] Vote for Release v0.6.1-rc0

### DISCUSSION THREAD

**Discussion by zhoujinsong** [apache/amoro#2544] [Polls]
Hi Everyone,

I propose that we release the following RC as the official Amoro 0.6.1 release.

The commit ID is https://github.com/NetEase/amoro/commit/e69c97b4705ebf2555c856790e781febaa613f73

This corresponds to the tag: v0.6.1-rc0
https://github.com/NetEase/amoro/commits/v0.6.1-rc0
https://github.com/NetEase/amoro/tree/v0.6.1-rc0
Please check out, build, and test.

Please vote in the next 72 hours and add your test steps when voting.

**Comment by xxubai** [comment]
+1 
I ran it in our environment and it works fine for iceberg compaction.

**Comment by zhongqishang** [comment]
+1

I have built and deployed AMS in our environment with jdk8, hadoop2, flink-1.14.

Tested the following features:

- Using MySQL as sysdb
- Create the optimizer group (external container) and start a flink optimizer
- Insert by flink cdc
- Table optimizing and result 
- Optimize task with large eq delete, and bloom filter works fine

**Comment by xieyi888** [comment]
+1
I have built and deployed AMS in our environment with jdk8, hadoop2, flink-optimizer-1.14.

Tested the following features:
ams working on HA mode
Using MySQL as sysdb
Create the optimizer group (external container) and start a flink optimizer container
mix_hive catalog, primary table, insert by flink cdc, spark read and result

**Comment by zhoujinsong** [comment]
The vote to release Amoro v0.6.1-rc0 has passed.
Thanks for reviewing and voting for our release candidate! @XBaith @zhongqishang @xieyi888 
I will be working on publishing the artifacts soon.
